runtime/frequency/src/lib.rs

```diff
@@ -788,6 +794,18 @@ impl pallet_frequency_tx_payment::Config for Runtime {
 	type MaximumCapacityBatchLength = MaximumCapacityBatchLength;
 }
 
+#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
```

Comment:
revisit for last review

Reply:
Some explanation here might be nice. Perhaps I'm reading it backwards, but it looks like these configs would be included for linting OR if we're building with the relay chain enabled... based on these values, I would think we would include them only for an instant-seal node... am I misreading something?

Reply:
I believe this is referring to this entire new section about preping for async backing

Reply:
These values are for setting up Async Backing. They are the values from the parachain-template.